runtime/store-tool/src/main.rs

```diff
@@ -0,0 +1,47 @@
+use clap::{crate_description, crate_name, value_t_or_exit, App, Arg};
+use log::*;
+use solana_runtime::append_vec::AppendVec;
+
+fn main() {
```

Comment:
well, there is never-merged similar pr: https://github.com/solana-labs/solana/pull/9490/files#diff-fe20bb9062954767913766205f72e8eb5ae88909c0d2529ec706e27a3e0eb800R589

Reply:
and?